UnitsNet.Tests/CustomCode/PressureTests.cs

```diff
@@ -88,6 +88,8 @@ public class PressureTests : PressureTestsBase
 
         protected override double MillimetersOfMercuryInOnePascal => 7.50061561302643e-3;
 
+        protected override double MeterOfWaterColumnInOnePascal => 1.0197162129779283e-4;
```

Comment:
Verified

1/( 98.0665 ) = 0.0101971621297792824257009274319
https://en.wikipedia.org/wiki/Centimetre_or_millimetre_of_water

0.000101971621297792824257009274318957034257366174993499309142265707453615658762* m.H2O
https://calculate.plus/en/categories/pressure/convert/Pa/to/meter-of-water-column?utm_content=cmp-true